less/common/LoadingIndicator.less

```diff
@@ -12,3 +12,50 @@
 .LoadingIndicator--block {
   height: 100px;
 }
+
```

Comment:
Is this the CSS-only solution? If so, it should be a separate PR.

Reply:
No, these are the styles from `spin.js`. Before, they were dynamically injected when the JS was loaded (or called?), but now they're split.

They either need to be included with `import "node_modules/spin.js/style.css"`, or this methods. I don't think our webpack setup allows for important stylesheets, so I just added it here, as it was the most logical place.

Reply:
Can we skip updating spin.js then? I'd rather revisit it as a whole in #2345